modelscope/pipelines/cv/dense_optical_flow_estimation_pipeline.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+from typing import Any, Dict, Union
+
+import cv2
+import numpy as np
+import PIL
+import torch
+
+from modelscope.metainfo import Pipelines
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Input, Model, Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.preprocessors import LoadImage
+from modelscope.utils.constant import Tasks
+from modelscope.utils.cv.image_utils import flow_to_color, InputPadder
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.dense_optical_flow_estimation,
+    module_name=Pipelines.dense_optical_flow_estimation)
+class DenseOpticalFlowEstimationPipeline(Pipeline):
+
+    def __init__(self, model: str, **kwargs):
```

Comment:
please refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L36 to add more docstr for pipeline